src/zenml/orchestrators/step_runner.py

```diff
@@ -531,8 +531,9 @@ def load_and_run_hook(
             output_materializers: The output materializers of the step.
         """
         try:
-            hook = source_utils.load_source_path(hook_source)
+            hook = source_utils.import_class_by_path(hook_source)
```

Comment:
Just out of curiosity, can you explain why this change necessary here?

Reply:
Good catch. Actually it might not be any more. I can probably just revert it back. I changed this early on when we were grabbing objects from globals and locals. I'll revert it and rerun the tests.

Reply:
Indeed the change was unnecessary with the later fix we made.